detox/src/devices/runtime/RuntimeDevice.js

```diff
@@ -419,6 +420,12 @@ class RuntimeDevice {
       this._currentApp.bundleId = await this.deviceDriver.getBundleIdFromBinary(binaryPath);
     }
   }
+
+  async resetAppState() {
+    await traceCall('resetAppState', () => {
+      return this.deviceDriver.resetAppState(this._currentApp.binaryPath, this._currentApp.bundleId);
+    });
+  }
```

Comment:
Please only place private methods in the end. Put this adjacent to something more associated with the same context.